### ISSUE TITLE
template - Move license URL to verification file

### ISSUE THREAD

**Issue by FranklinYu** [chocolatey/choco#1950]
It is proposed to move the license URL from `LICENSE.txt` to `VERIFICATION.txt`. Reasons:
* Most (if not all) core team packages are already placing the URL in verification file.
* It makes sense not to modify 3rd party files. Many software already provides `LICENSE.txt` as a file.

This two reasons has been supported by two member and two contributors in chocolatey-community/chocolatey-coreteampackages#1294. Affected source folder:

https://github.com/chocolatey/choco/blob/master/src/chocolatey/infrastructure.app/templates/

**Comment by ferventcoder** [comment]
The URL doesn't need to be there - the template should be updated to have that there as a template for those who don't know what they are looking to do - for those that know better, they know they can simply replace the entire file.

**Comment by FranklinYu** [comment]
@ferventcoder Excuse me but were you referring to `LICENSE.txt` or `VERIFICATION.txt`?

**Comment by ferventcoder** [comment]
Was referring to the license file.

**Comment by FranklinYu** [comment]
Although the license URL can be in either file, now that most maintainers prefer to put it in `VERIFICATION.txt`, I think it makes more sense to update those two templates in order not to confuse new maintainers; for example, I was seriously questioned about my decision to put URL in `VERIFICATION.txt`. It seems good to have the template reflects consensus among active maintainers and administrators.

**Comment by pauby** [comment]
As the license location is already in the `<licenseUrl>` field in the .nuspec file, do we need to specify it again? 

In the cases where the license doesn't exist at a URL (for example it's embedded in the Zip or even shown in the installer but not available anywhere else) a note could be added either to the VERIFICATION.txt or the `<description>` (whichever one of those we choose we should stick to so moderators don't have to go searching for it in both places). In absence of any note the moderators would assume the contents of the LICENSE.txt should match the license found at `<licenseUrl>`?

**Comment by FranklinYu** [comment]
> As the license location is already in the `<licenseUrl>` field in the .nuspec file, do we need to specify it again?

I agree with you that this isn't necessary. I did this merely because everyone else is doing this. I would be more than happy if we remove this requirement altogether and all moderators simply look at license URL in nuspec since they are structured and clickable.

> In absence of any note the moderators would assume the contents of the LICENSE.txt should match the license found at `<licenseUrl>`?

Yes exactly.

> In the cases where the license doesn't exist at a URL (for example it's embedded in the Zip or even shown in the installer but not available anywhere else) a note could be added either to the VERIFICATION.txt or the `<description>` (whichever one of those we choose we should stick to so moderators don't have to go searching for it in both places).

I think we should settle it down in this thread. I myself vote for `VERIFICATION.txt`. Reason: description is for users, while `VERIFICATION.txt` is (typically) for feed moderators.
